library/item/item_group/src/main/java/org/quiltmc/qsl/item/group/mixin/client/CreativeInventoryScreenMixin.java

```diff
@@ -165,6 +174,19 @@ private void renderTabIcon(MatrixStack matrixStack, ItemGroup itemGroup, Callbac
 		if (this.quilt$isGroupNotVisible(itemGroup)) {
 			info.cancel();
 		}
+
+		this.quilt$renderTabIcon$matrices = matrixStack;
```

Comment:
Once again - no reason to capture these to a field. Just capture them in the `Redirect` below (or preferably a `ModifyArgs`)

Reply:
Eh so I could use a local capture but first off that can't capture the matrices, second off `renderTabIcon` has about 8 locals that I don't want to filter through.

Reply:
No, not a local capture. Just capture the method args, right?

Reply:
ah

Reply:
Resolved.